src/modules/core/components/InfoPopover/TokenInfo.tsx

```diff
@@ -48,7 +47,7 @@ const TokenInfo = ({ token, isTokenNative }: Props) => {
           </p>
         )}
         <div title={address} className={styles.address}>
-          <CopyableAddress full>{address}</CopyableAddress>
+          {address}
```

Comment:
Are we sure that we dont make this copyable?

Reply:
Yeah... Now that I think more about it, I don't see why not. I removed it initially because in the design it doesn't really look copyable